tests/test_log_artifact.py

```diff
@@ -67,45 +67,45 @@ def test_log_artifact_with_save_dvc_exp(tmp_dir, mocker, mocked_dvc_repo):
 def test_log_artifact_type_model(tmp_dir, mocked_dvc_repo):
     (tmp_dir / "model.pth").touch()
 
-    with Live() as live:
+    with Live(dvcyaml="dvc.yaml") as live:
```

Comment:
Shouldn't this be the default behavior when no argument is passed? Why do we need to explicitly pass it in the test?